content/sdk/80.troubleshooting.md

```diff
@@ -8,61 +8,67 @@ navigation: false
 Whether you’re facing integration challenges, encountering unexpected errors, or need clarity on certain functionalities,
 here are detailed solutions and best practices to ensure a smooth development experience.
 
-::card-group
-::card
----
-title: JavaScript
-icon: i-heroicons-code-bracket
-to: #
----
-Troubleshooting JavaScript SDK Issues.
-::
-::card
----
-title: GO
-icon: i-heroicons-cog
-to: #
----
-Troubleshooting Go SDK Issues.
-::
-::card
----
-title: Python
-icon: i-heroicons-circle-stack
-to: #
----
-Troubleshooting Python SDK Issues.
-::
-::card
----
-title: Java
-icon: i-heroicons-rectangle-group
-to: #
----
-Troubleshooting Java SDK Issues.
-::
-::card
----
-title: Swift
-icon: i-heroicons-rocket-launch
-to: #
----
-Troubleshooting Swift SDK Issues.
-::
-::card
----
-title: Rust
-icon:  i-heroicons-wrench
-to: #
----
-Troubleshooting Rust SDK Issues.
-::
-::card
----
-title: Bun
-icon: i-heroicons-cube
-to: #
----
-Troubleshooting Bun Issues.
-::
-::
+## JavaScript SDK
+
+  ::collapsible
+    :field If you encounter issues using Js SDK during the installation, refer to the troubleshooting steps:
```

Comment:
This is not the correct format for the `:field` component. 
`:field{name=""}`
But this also doesn't seem to be an appropriate use of the field component for this bit of text.

Reply:
The issue here is that `:field` you're using isn't valid so it's not even formatting anything correctly with that.